src/main/java/uk/gov/digital/ho/hocs/document/MsgContents.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.digital.ho.hocs.document;
+
+import com.auxilii.msgparser.attachment.Attachment;
+import lombok.Getter;
+import lombok.Setter;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.List;
+
+@Getter
+@Setter
+public class MsgContents {
```

Comment:
Class with no proper constructor but with setters for all its private variables.

I think we should use all args constructor, or even better a constructor that takes in Message directly:
MsgContents(Message message){
...
}

Reply:
Constructor is not required as it fields are set separately.